build-wheezy.sh

```diff
@@ -4,7 +4,7 @@ sudo apt-get update
 
 sudo apt-get --yes --force-yes install git cmake gcc-4.8 g++-4.8 automake autoconf libtool libboost1.50-dev libboost-system1.50-dev libboost-thread1.50-dev libboost-program-options1.50-dev libboost-filesystem1.50-dev
 
-git clone --depth=1 git://sigrok.org/libserialport
+git clone --depth=1 https://github.com/martinling/libserialport
```

Comment:
hum, I'm wondering why that changed since I used a clean stat of the ship branch of upstream... not the end of the word though, but may want to check which source should be used for libserialport (that's the one I used myself, right or wrong).